src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -1593,12 +1593,6 @@ public bool IsLogicalOrExpression(SyntaxNode node)
         public bool IsLogicalNotExpression(SyntaxNode node)
             => node.Kind() == SyntaxKind.LogicalNotExpression;
 
-        public bool IsConditionalAnd(SyntaxNode node)
-            => node.Kind() == SyntaxKind.LogicalAndExpression;
-
-        public bool IsConditionalOr(SyntaxNode node)
-            => node.Kind() == SyntaxKind.LogicalOrExpression;
```

Comment:
returns teh same as IsLogicalAnd and IsLogicalOr (defined a few lines above at https://github.com/dotnet/roslyn/pull/41525/files#diff-1eda1d67f1b7acf2ea04b9af6d0e105aR1587).